include/extractor/original_edge_data.hpp

```diff
@@ -24,7 +24,7 @@ struct GeometryID
     {
     }
 
-    GeometryID() : id(std::numeric_limits<unsigned>::max()), forward(false)
+    GeometryID() : id(std::numeric_limits<unsigned>::max() >> 1), forward(false)
```

Comment:
Can we get a constant for this defined somewhere? If its an arbitrary value apart from 0, max, it is easy to mix it up in checks.